driver/http.go

```diff
@@ -669,12 +669,18 @@ func (c *httpCRUD) deleteWithOptions(ctx context.Context, collection string, fil
 
 func (c *httpCRUD) readWithOptions(ctx context.Context, collection string, filter Filter, fields Projection, options *ReadOptions) (Iterator, error) {
 	ctx = setHTTPTxCtx(ctx, c.txCtx, c.cookies)
-
+	var sort []json.RawMessage
+	if options.Sort != nil {
+		sort = []json.RawMessage{json.RawMessage(options.Sort)}
```

Comment:
I think we need to fix the HTTP client to accept just json.RawMessage.
To do that you can change `yq_fix_json ReadRequest sort.items`
to `yq_fix_json ReadRequest sort` in scripts/fix_openapi.sh
and run `make generate`.